.github/workflows/pr.yaml

```diff
@@ -29,10 +43,6 @@ jobs:
         # Confirm that we get a response on port 9000.
         run: while ! docker run --network host appropriate/curl -v -s --retry-max-time 180 --retry-connrefused http://localhost:9000/ ; do sleep 5; done
         timeout-minutes: 3
-      - name: Specifically test that webpack assets are present.
-        # Confirm that we get a response on port 9000.
-        run: while ! docker run --network host appropriate/curl -v -s --retry-max-time 180 --retry-connrefused http://localhost:9000/assets/javascripts/bundle.js ; do sleep 5; done
-        timeout-minutes: 1
```

Comment:
We don't use webpack anymore, so I'm surprised this has been passing.

Reply:
Does the js file still wind up at bundle.js? That would explain it.

Reply:
Maybe just cached? It was sitting around on my local dev file system. I deleted it and re-ran `docker build -f prod.Dockerfile -t civiform:prod ./` and `docker-compose -f infra/prod-simulator-compose.yml up` and it didn't regenerate.

Reply:
Hmmm... shouldn't be, but all right, seems like a pretty benign mystery.  I'll investigate while tests are running or I'm otherwise sitting around but won't freak out if I don't figure it out.